library/core/src/result.rs

```diff
@@ -335,16 +347,24 @@
 //! * [`map`] transforms [`Result<T, E>`] into [`Result<U, E>`] by applying
 //!   the provided function to the contained value of [`Ok`] and leaving
 //!   [`Err`] values unchanged
+//! * [`inspect`] takes ownership of the [`Result`], applies the
+//!   provided function to the contained value by reference,
+//!   and then returns the [`Result`]
 //!
 //! [`map`]: Result::map
+//! [`inspect`]: Result::inspect
 //!
 //! This method transforms the contained value of the [`Err`] variant:
```

Comment:
```suggestion
//! These methods transform the contained value of the [`Err`] variant:
```

Same with the `Ok` one about, I can't add a review comment on it directly.